packages/components/src/components/common/dialog/dialog.css.ts

```diff
@@ -54,7 +54,9 @@ export const content = style({
   padding: '20px',
   backgroundColor: vars.color.background,
   border: vars.border.basic,
-  display: 'grid',
+  maxHeight: '80vh',
+  display: 'flex',
```

Comment:
This change can break existing Dialogs.
Not against it, but I think you have to check every dialog if everything still works as expected.

Maybe it is easier to just change the maxHeight of the basic dialog content?